templates/webapi/comments/build_pagination.html.ep

```diff
@@ -0,0 +1,88 @@
+<div class="row comment-row">
+    <div class="col-sm-1"></div>
+    <div class="col-sm-11">
+        <nav class="comments-pagination <%= $media_class %>" aria-label="Comments pagination">
+            <ul class="pagination">
+            % if ($comments_pager->current_page == $comments_pager->first_page) {
+                <li class="disabled"><span
+            % } else {
+                <li><a href="<%= url_with->query([ comments_page => $comments_pager->current_page - 1 ]) %>"
+            % }
+                class="page-link" aria-label="Previous">
+                  <span aria-hidden="true">&laquo;</span>
+                  <span class="sr-only">Previous</span>
+              % if ($comments_pager->current_page == $comments_pager->first_page) {
+                  </span>
+              % }
+              % else {
+                  </a>
+              % }
+
+              % if ($comments_pager->last_page <= $max) {
+                  % for (my $i = 1; $i <= $max; $i++){
+                      %= tag 'li', class => ($i == $comments_pager->current_page ? 'active' : undef) => sub {
+                      %    link_to $i => url_with->query([ comments_page => $i ]) => (class => 'page-link')
+                      % };
+                  %}
+              % } else {
+                  %= tag 'li', class => (1 == $comments_pager->current_page ? 'active' : undef) => sub {
+                  %    link_to 1 => url_with->query([ comments_page => 1 ]) => (class => 'page-link')
+                  % };
+
+                  % my $surround = int(($max - 3) / 2);
+                  % my $from = $comments_pager->current_page - $surround;
+                  % my $to =  $comments_pager->current_page + $surround;
+
+                  % if ($from <= 1) {
+        	            % $to += ( -1 * $from + 2 );
+        				% $from = 2;
```

Comment:
Don't mix tabs and spaces (we're using spaces). I understand that this is still a draft. However, we don't have automatic formatting for templates so it makes sense to setup your editor correctly in the first place.